interface-definitions/system-event-handler.xml.in

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<interfaceDefinition>
+  <node name="system">
+    <children>
+      <tagNode name="event-handler" owner="${vyos_conf_scripts_dir}/system_event_handler.py">
```

Comment:
What do you think of moving it under a separate node, like we do with firewall now? `set system event-handler event Foo pattern "foo(.*)"`.

Reply:
Moved to
```
set service event-handler Foo
```

Reply:
done